js/builder/build.js

```diff
@@ -129,9 +129,9 @@ class Build{
                 }
             }
         }
-        statMap.set("poisonPct", 100);
-        statMap.set("critDamPct", 100);
-        statMap.set("healPct", 100);
+        statMap.set("poisonPct", 0);
+        statMap.set("critDamPct", 0);
+        statMap.set("healPct", 0);
```

Comment:
Implementation note: These were changed to default to 0, and 1+x/100 to support radiance more easily.

also realizing that healpct might be handled incorrectly